.eslintignore

```diff
@@ -7,3 +7,5 @@
 CHANGELOG.md
 dist
 node_modules
+/test/regressions/build
```

Comment:
This should be enough. It's the folder name we use as a convention for auto generated code. 

```suggestion
build
```

See https://github.com/mui-org/material-ui/blob/a1b75ccf8edcf7a58dacca7e6b1b1e1328e32409/.eslintignore#L25